aexpect/ops_linux.py

```diff
@@ -432,6 +432,29 @@ def grep(session, expr, path, check=False, flags="-a"):
     return output
 
 
+def grep_pipe(session, cmd, pattern, flags = "", check = False):
+    """
+    Invoke ``grep`` remotely searching for an expression in a command output.
+
+    :param session: vm session
+    :type session: ShellSession
+    :param str cmd: command that its output will be grepped
+    :param str pattern: pattern to grep
```

Comment:
Could you please unify this with the `grep` param above? (`expr` rather than `pattern`)